guides/v2.1/cloud/cdn/cloud-fastly-custom-response.md

```diff
@@ -0,0 +1,159 @@
+   - /guides/v2.1/cloud/cdn/configure-fastly.html#fastly-errpg
+   - /guides/v2.2/cloud/cdn/configure-fastly.html#fastly-errpg
+   - /guides/v2.3/cloud/cdn/configure-fastly.html#fastly-errpg
+
+functional_areas:
+  - Cloud
+  - Setup
+  - Configuration
+---
+
+When a request to the Fastly origin fails, Fastly returns default response pages with basic formatting and generic messaging that can be confusing for users. For example, Fastly returns the following default error page when an origin request fails with a 503.
+
+![Fastly default error page]
+
+You can update your {{ site.var.data.ee }} store configuration to replace some default response pages with pages that have friendlier messaging and improved HTML styling as shown in the following example.
+
+![Fastly custom error page]
+
+Currently, you can customize the following Fastly response pages for your {{ site.data.var.ece }} project through the Magento Admin UI.
+
+- [Timeout or site maintenance outages (503 Service Unavailable)](#customize-the-503-response-page)
+- [WAF blocking events that occur when the WAF detects suspicious request traffic (403 Forbidden)](#customize-the-waf-response-page)
+
+
+**HTML coding requirements**
+
+The HTML code for the custom page must meet the following requirements:
+
+- Content can contain up to 65,535 characters.
+- Specify all CSS inline in the HTML source.
+- Bundle images in the HTML page using base64 so that they display even if Fastly is offline. See [Data URIs on the css-tricks site](https://css-tricks.com/data-uris/).
+
+
+## Customize the 503 response page
+
+Customers see the default 503 response page in the following cases:
+
+- When a request to the Fastly origin returns a response status great than 500.
+- When the Fastly origin is down, for example due to a timeout, maintenance activity, or health issues.
+
+You can customize the default page by adapting the following HTML code to include styling to match your {{ site.data.var.ee }} store theme and modifying the title and messaging as needed.
+
+```html
+<!DOCTYPE html>
+<html>
+   <head>
+      <meta charset="UTF-8">
+         <title>503</title>
+   </head>
+   <body>
+      <p>Service unavailable</p>
+   </body></html>
+```
+
+Verify that the modified source displays correctly in the browser. Then, add the customized HTML code to the Fastly configuration.
+
+### Add the custom response page to the Fastly configuration
+
+1.  [Log in to the Magento Admin UI]({{ page.baseurl }}/cloud/onboarding/onboarding-tasks.html#admin).
+
+1.  Click **Stores > Settings > Configuration > Advanced > System**.
+
+1.  In the right pane, expand **Full Page Cache > Fastly Configuration > Custom Synthetic Pages**.
+
+    ![Edit 503 error page]
+
+1.	Click **Set HTML**.
+
+1.	Copy and paste the source code for your custom response page into the HTML field. 
+
+    ![Update 503 response]
+	
+1.	Click **Upload** to update the Fastly configuation.
+
+1.	Click **Save Config** at the top of the page.
```

Comment:
Upload was at the top of the page and you didn't mention it. Is it important to note the location of Save Config? Would they be confused if you did or didn't? No need to change if you feel it is important, I'm just thinking aloud.